### ISSUE TITLE
And for plain faster rcnn, when I use my own faster rcnn, and faster rcnn implemented by you to fit my dataset with the same parameter setting, the results are different and have a big difference. Do you think I should set different parameters? Do you have any advice about that? Thanks

### ISSUE THREAD

**Issue by hdjsjyl** [bowenc0221/Decoupled-Classification-Refinement#11]
Hi, thanks for your advice, finally, I get your results. But I am confused, for my faster rcnn and your plain faster rcnn. I need to use different parameters setting to get the same results on my own dataset. Can you provide some advice about that? thanks

**Comment by bowenc0221** [comment]
I guess one difference might be how to set learning rate: https://github.com/bowenc0221/Decoupled-Classification-Refinement/blob/9c37d014212f2e9f335a7262e3b1c24cd6774e49/faster_rcnn/train_end2end.py#L152
`rescale_grad ` is `1/NUM_GPUS` by default, but I set it to `1.0`. So in this implementation, the actual learning rate is lr * NUM_GPUS.

**Comment by hdjsjyl** [comment]
thanks for your reply. I got it. The difference of weight decay of 0.0005 and 0.0001 have big influence on final performance? Thanks

**Comment by bowenc0221** [comment]
Yes, weight decay has a very large influence. You should not change it.

**Comment by hdjsjyl** [comment]
thank you. Why it has a big influence?
